nlu_hyperopt/app.py

```diff
@@ -53,7 +69,14 @@ def worker_function(space):
     best_config = space_eval(search_space, best)
     logger.debug("The best values are: {}".format(best_config))
 
-    data_dir = os.environ.get("DATA_DIRECTORY", "/data")
+    data_dir = os.environ.get("INPUT_DATA_DIRECTORY", "./data")
```

Comment:
mhm. What if the provided paths are relative (e.g. `/data`) but we are in a GitHub action? Should we prepend the working directory then? 🤔

Reply:
I figured people would just explicitly say their path was relative to `GITHUB_WORKSPACE`, but that actually makes sense (since what else would it be relative to?)

Reply:
They should specify it as `./data` right? Even if it is relative. That works  after taking out the `WORKDIR` line from the dockerfile

Reply:
mhm, why don't you do:

```
data_dir = Path(data_dir)
if running_as_action and not data_dir.is_absolute():
  data_dir = Path("mounted volume path) / data dir
```

Reply:
mhm, but if it works already then it's probably fine 👍